istio/install/istio-noauth.yaml

```diff
@@ -0,0 +1,5159 @@
+apiVersion: v1
```

Comment:
Why noauth? Does "auth" refer to RBAC or sidecar?

Reply:
noauth means no mutual TLS between service